sched/scheduler/cluster_state.go

```diff
@@ -290,13 +292,13 @@ func (c *clusterState) update(updates []cluster.NodeUpdate) {
 			// This node is initialized, remove it from suspended nodes and add it to the healthy node pool.
 			c.nodes[ns.node.Id()] = ns
 			delete(c.suspendedNodes, ns.node.Id())
-			log.Infof("Node now ready, adding to rotation: %v (%#v), %s", ns.node.Id(), ns, c.status())
+			log.Infof("Node now ready, adding to rotation: %v (%s), %s", ns.node.Id(), ns, c.status())
 		} else if ns.timeLost != nilTime && now.Sub(ns.timeLost) > c.maxLostDuration {
 			// This node has been missing too long, delete all references to it.
 			delete(c.suspendedNodes, ns.node.Id())
 			delete(c.nodeGroups[ns.snapshotId].idle, ns.node.Id())
 			delete(c.nodeGroups[ns.snapshotId].busy, ns.node.Id())
-			log.Infof("Deleting lost node: %v (%#v), %s", ns.node.Id(), ns, c.status())
+			log.Infof("Deleting lost node: %v (%s), %s", ns.node.Id(), ns, c.status())
```

Comment:
Think we still want %#v on ns, and the first %v can be a %s for ns.node instead of ns.node.Id()

Reply:
fixed